auslib/blobs/gmp.py

```diff
@@ -61,6 +61,15 @@ def getInnerXML(self, updateQuery, update_type, whitelistedDomains, specialForce
 
         return vendorXML
 
+    def validate(self, *args, **kwargs):
+        Blob.validate(self, *args, **kwargs)
+        hashes = dict(SHA1=40, SHA256=64, SHA512=128)
```

Comment:
Yeah, that does. I guess I just move out the code, make a function, make the call **here** itself, or should it be called somewhere else. Any particular place, where this function belongs? I'd suggest `Blob` itself :thinking: ?